spec/alephant/broker/load_strategy/http_spec.rb

```diff
@@ -4,6 +4,7 @@
   subject { described_class.new(url_generator) }
 
   let(:component_meta) do
+    # FIXME: ComponentMeta doesn't have a #cache_key method???
```

Comment:
It's a new one from me... I'm not really sure what to do about it as I can't see _how_ the code is actually working in production as it's calling an undefined method.

I can fix it, but might need some hand holding getting something live to slurp in the change and make sure nothing breaks